README.md

```diff
@@ -566,6 +566,19 @@ import { addMethod, Schema } from 'yup';
 addMethod(Schema, 'myMethod', ...)
 ```
 
+
+You can add your test methods to the schema using `addMethod` by returning `this.test()`.  This will allow you to reuse them as well. 
```

Comment:
Rather than add add a new section here, let's add the code example to the first code block and then demonstrate how to use the new added methods. For extra clarity i think it's worth adding a few words about `this` being the the schema instance, and so any of the methods documented in the docs will be on them.

Reply:
Sure.  I will make the change and resubmit.